<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color: black;
                       background-color:rgb(200, 200, 200);   
border-radius:5px;                     /* &lt;----  20px  */ 
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выберите товар :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: black;
                       background-color:rgb(240,240,240);   
border-radius: 5px;                    
border: 2px solid #094065;
</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>411</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
                       background-color:rgb(240,240,240);   
border-radius: 5px;                    
border: 2px solid #094065;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Текущий(е) товар(ы) :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Общая сумма чека :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLineEdit {
color: black;
                       background-color:rgb(240,240,240);   
border-radius: 5px;                    
border: 2px solid #094065;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{color: black;
                       background-color:rgb(240,240,240);   
border-radius: 5px;                     /* &lt;----  20px  */ 
border: 2px solid #094065;
                                      
}
QPushButton:hover {
 background:rgb(202, 219, 255);
transition: 1s;
}
</string>
   </property>
   <property name="text">
    <string>Продать</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/point_of_sale_FILL0_wght400_GRAD0_opsz48.svg</normaloff>../icons/point_of_sale_FILL0_wght400_GRAD0_opsz48.svg</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Количество :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: black;
                       background-color:rgb(240,240,240);   
border-radius: 5px;                    
border: 2px solid #094065;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>400</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{color: black;
                       background-color:rgb(240,240,240);   
border-radius: 5px;                     /* &lt;----  20px  */ 
border: 2px solid #094065;
                                      
}
QPushButton:hover {
 background:rgb(202, 219, 255);
transition: 1s;
}
</string>
   </property>
   <property name="text">
    <string>Удалить товар из чека</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/delete_FILL0_wght400_GRAD0_opsz48.svg</normaloff>../icons/delete_FILL0_wght400_GRAD0_opsz48.svg</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{color: black;
                       background-color:rgb(240,240,240);   
border-radius: 5px;                     /* &lt;----  20px  */ 
border: 2px solid #094065;
                                      
}
QPushButton:hover {
 background:rgb(202, 219, 255);
transition: 1s;
}
</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/add_FILL0_wght400_GRAD0_opsz48.svg</normaloff>../icons/add_FILL0_wght400_GRAD0_opsz48.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
